blueocean-core-js/src/js/capability/Capability.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Created by cmeyers on 9/8/16.
+ */
+export const CAPABILITY = {
+    SIMPLE_PIPELINE: 'org.jenkinsci.plugins.workflow.job.WorkflowJob',
```

Comment:
Actual capability references should not be defined in this core module. We should come up with an easier way to reference them... my suggestion, which isn't perfect, is to add a `<obj>._isa.<ShortClassName/CapabilityName>` or `_can` so we don't have to define a bunch of random stuff all over.

For example, if we had a capability that was `displayTestResults`, it would be mapped to pertinent objects as `<obj>._can.displayTestResults` or... `<obj>._can_displayTestResults` so we don't have to test for `_can` all over the place. Similarly classes might be mapped by short class name, so the above example would map to: `<obj>._isa.WorkflowJob` or `<obj>._isa_WorkflowJob`. We could optimize some property adders to closures and reuse JS strings for this stuff, if necessary.

It would be nice to have the class info and capability info separate for `_isa` and `_can` references, respectively.